config.json

```diff
@@ -1430,6 +1430,18 @@
         "strings"
       ],
       "deprecated": true
+    },
+    {
+      "slug": "grep",
+      "uuid": "78bcbae1-a0f2-460c-ba89-e51844fe9397",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
Let's pick an exercise. Just pick an exercise with `"core": true`, where you think it would be nice to have this as a side _after_! It can be any of the core exercises.

Reply:
Alright, I'll try that, but I don't know all the exercises, so i'll choose given the difficulty of the core exercises